pos_frequency_pipeline.py

```diff
@@ -13,15 +18,45 @@ class EmptyFileError(Exception):
 
 class POSFrequencyPipeline:
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
-        pass
+        for article in self.corpus_manager.get_articles().values():
+            frequencies = {}
+            pattern = re.compile(r'<([A-Z]+)')
+            with open(article.get_file_path('single_tagged'), 'r') as file:
```

Comment:
the same thing goes here: instead of passing a plain string, use ArtifactType attribute

Reply:
I fixed that too